docs/src/main/sphinx/installation/containers.md

```diff
@@ -88,6 +88,18 @@ at `/etc/trino/catalog`, or individual catalog property files in it.
 
 If you want to use additional plugins, mount them at `/usr/lib/trino/plugin`.
 
+To avoid having to create catalog files and mount them in the container,
+dynamic catalog management can be enabled by setting the `CATALOG_MANAGEMENT`
```

Comment:
```suggestion
you can enable dynamic catalog management by setting the `CATALOG_MANAGEMENT`
```

Reply:
also this only works if you don't provide your our mount for the config. If you do then the variable is simply ignored.

Reply:
Why it would be ignored? We don't change the catalog store type, it defaults to file and trino should read catalog property files on startup with dynamic catalogs enabled

Reply:
Oh you mean the main config. I can add a check for that and throw an error

Reply:
yes but my point is why introduce this complexity when it can be better solved by having a guide to show users how to add tweak config.properties. Surely `catalog.management` is not the only config people will want to change? If someone wants to change any other config they'll HAVE to mount a config file.

I'm fine with treating `CATALOG_MANAGEMENT` as a special case but just want us to be aware that it doesn't help in cases other than just doing a quick dynamic-catalog specific demo for example.

Reply:
That was my initial concern as well. And we already have a section that talks about custom config file usage in the docs. Maybe we can just add an example that sets up the dynamic catalog usage and not worry about the parameter and all that complexity.

Reply:
https://trino.io/docs/current/installation/containers.html#configuring-trino

Also not I have a similar example documented now for OpenTelemetry where I document how to mount the custom config for it.

https://trino.io/docs/current/admin/opentelemetry.html#example-use

Reply:
Mounting files is not easy, having examples doesn't help since you have to create multiple new files locally. A single command is always better than multiple steps.

My only intention here is to make running Trino as easy as possible, not solving general configuration issues. I understand the risks, but I think they're acceptable.

The best solution is having dynamic catalogs enabled by default, but we don't know when that'll be possible. I don't think anyone is working on solving the security issues.

If you're both against this, please close this PR. I don't feel strongly about this.

Reply:
I think this usecase is important enough to use this approach.